packages/dataviews/src/pagination.js

```diff
@@ -27,36 +27,33 @@ const Pagination = memo( function Pagination( {
 				justify="end"
 				className="dataviews-pagination"
 			>
-				<HStack justify="flex-start" expanded={ false } spacing={ 2 }>
-					{ createInterpolateElement(
-						sprintf(
+				<HStack justify="flex-start" expanded={ false } spacing={ 1 }>
+					<SelectControl
+						aria-label={ __( 'Current page' ) }
```

Comment:
Instead of `aria-label`, could we use the `label` prop + the `hideLabelFromVision` prop?